usr/src/cmd/dd/dd.c

```diff
@@ -1839,6 +1876,13 @@ static unsigned char
 				"wrote %d bytes, expected %d\n"), bc, oc);
 			term(2);
 		}
+
+		if (ostriden && lseek(obf, (((off_t)ostriden) * ((off_t)obs)),
```

Comment:
See above.  And no, not as far as I can tell. They should both be using >. Probably that happened when I added istride after ostride. Sorry!